assets/js/base/utils/test/price.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Internal dependencies
+ */
+import { formatPrice } from '../price';
+
+describe( 'formatPrice', () => {
+	test.each`
+		value        | priceFormat   | currencySymbol | expected
+		${10}        | ${'%1$s%2$s'} | ${'€'}         | ${'€10'}
+		${10}        | ${'%2$s%1$s'} | ${'€'}         | ${'10€'}
+		${10}        | ${'%2$s%1$s'} | ${'$'}         | ${'10$'}
```

Comment:
Nice!  Thanks for adding this test - good practice to have all our utilities covered by tests so I appreciate this :)